clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -1216,6 +1254,7 @@ public static class ConfigKey {
         public final List<String> dependents;
         public final Recommender recommender;
         public final boolean internalConfig;
+        public final Object overwrittenValue;
```

Comment:
Can we find a better name? Or at least add a comment to explain what this is?

Given what we use `overwrittenValue` to generate HTML only, should it be `String` type?

Reply:
Renamed it to `alternativeString` and change it to String type